components/financial_assistance/db/seedfiles/translations/en/me/financial_assistance.rb

```diff
@@ -184,9 +184,13 @@
   "en.faa.update_reminder" => "If you make changes to your %{year} coverage that should apply to your %{year2} coverage too, you will need to come back and update your %{year2} application. You can do this by copying your latest %{year} application and submitting it for %{year2}. ",
   "en.faa.learn_more_about_life_changes" => "Learn more about life changes",
   # Mec check
-  "en.faa.mc_success" => "It looks like you may already be enrolled in MaineCare or Cub Care. If you need to update information like your income, address, or who is in your household, contact the Office for Family Independence at <a href='tel:855-797-4357'>(855) 797-4357</a> to make these changes before completing a CoverME.gov application.",
+  "en.faa.mc_success_existing" => "It looks like you may already be enrolled in MaineCare or Cub Care. If you need to update information like your income, address, or who is in your household, contact the Office for Family Independence at <a href='tel:855-797-4357'>(855) 797-4357</a> to make these changes before completing a CoverME.gov application.",
```

Comment:
Can try and pull out the telephone anchor tags, left them in because the only change to this was it's translation name.

Reply:
Similar to the above comment, let's use the Resource Registry setting that holds this number already and pass it in as an argument to the translation helper.  You may need to add an additional setting for the href formatted number or add/look for a helper method to convert the existing string to the correct format.